src/dotnet/Directory.Build.props

```diff
@@ -110,58 +110,5 @@
     <Watch Remove="**\*.js" />
     <Watch Remove="**\*.css" />
   </ItemGroup>
-
-  <!-- Extra trimming roots -->
-  <ItemGroup>
```

Comment:
Won't it affect Android build?
Or we use their partial trimming and all assebmlies are considered to not be trimmed if it's not specified explicitly?

Reply:
Will test tomorrow, but we use partial trimming and our assemblies including stl are not marked with trim attribute explicitly